PrimeNumberTheoremAnd/ResidueCalcOnRectangles.lean

```diff
@@ -237,7 +501,16 @@ The rectangle (square) integral of $f(s) = 1/s$ with corners $-1-i$ and $1+i$ is
 %%-/
 lemma ResidueTheoremAtOrigin :
     RectangleIntegral' (fun s ↦ 1 / s) (-1 - I) (1 + I) = 1 := by
-  sorry
+  dsimp [RectangleIntegral', RectangleIntegral]
+  rw [ResidueTheoremAtOrigin_aux1]
+  rw [add_sub_assoc]
+  have := ResidueTheoremAtOrigin_aux2
+  rw [ResidueTheoremAtOrigin_aux2]
+  have : (2 * π * I) ≠ 0 := by
+    norm_num
+    exact pi_ne_zero
+  field_simp
+  ring
 /-%%
 \begin{proof}
```

Comment:
Now you can add `\leanok` after `\begin{proof}` because you proved it! (That'll update the color in the blueprint dependency graph) Woohoo!!